### PR TITLE
Mr 70 product page implementation

### PR DESCRIPTION
**Pull request by JaqKent**
first part of the product page implementation

### CODE REVIEW

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Some comments before moving on

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Some comments before moving on

File: src/components/CustomButton/index.tsx [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
-      </button>
-    </div>
-  );
+    return (
+        <div className={className}>
+            <button
+                disabled={disabled}
+                className={`${small ? styles.small : styles.big} ${
+                    styles.primaryButton
+                } ${secondary ? styles.secondaryButton : styles.primaryButton}`}
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/CustomButton/index.tsx:28]: done

File: src/components/CustomButton/styles.module.scss [hunk 2]
```diff
@@ -56,15 +53,10 @@
 
 .big {
     @include flex(center, center, row);
-    border-radius: 4px;
+    border-radius: 15px;
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/CustomButton/styles.module.scss:56]: done

File: src/components/ProductPage/components/SelectDropdown/index.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-param-reassign */
+import React, { useState } from 'react';
+
+import { OPTIONS } from '../constants';
+
+import styles from './styles.module.scss';
+
+function SelectDropdown() {
+    const [selectedOption, setSelectedOption] = useState<string>();
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/ProductPage/components/SelectDropdown/index.tsx:9]: done

File: src/components/ProductPage/components/SelectDropdown/index.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-param-reassign */
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/ProductPage/components/SelectDropdown/index.tsx:1]: removed

File: src/components/ProductPage/components/SelectDropdown/index.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import React, { useState } from 'react';
+
+import { OPTIONS } from '../constants';
+
+import styles from './styles.module.scss';
+
+function SelectDropdown() {
+    const [selectedOption, setSelectedOption] = useState<string>();
+
+    const selectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/ProductPage/components/SelectDropdown/index.tsx:11]: done

File: src/components/ProductPage/components/StarRating/index.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+interface Props {
+    initialValue: number;
+    onClick?: () => void;
+    readonly?: boolean;
+    size: number;
+}
+
+function StarRating({ initialValue, readonly, onClick, size }: Props) {
+    const [ratingValue, setRatingValue] = useState(0);
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/ProductPage/components/StarRating/index.tsx:12]: the component use the state when the user click on the "star" for puntuate the product

File: src/components/ProductPage/components/quantitySelector/index.tsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    const [quantity, setQuantity] = useState(0);
+    const add = () => {
+        setQuantity(quantity + 1);
+    };
+    const subtract = () => {
+        setQuantity(quantity - 1);
+    };
+    const getFormattedQuantity = () => {
+        return quantity.toString().padStart(2, '0');
+    };
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/ProductPage/components/quantitySelector/index.tsx:15]: done

File: src/components/ProductPage/index.tsx [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+    const handleAddToCart = () => console.log('cart');
+    const handleBuyNow = () => console.log('buy now');
+
+    return (
+        <div className={styles.container}>
+            <div className={styles.title}>
+                <div className={styles.text}>
+                    Text wieght 400 align left poppin Text wieght 400 align left
+                    poppinText
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/ProductPage/index.tsx:24]: ok , i just use the placeholder from the design

File: src/components/Slider/index.tsx [hunk 9]
```diff
@@ -1,105 +1,130 @@
 /* eslint-disable no-unused-vars */
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/Slider/index.tsx:1]: removed

File: src/components/CustomButton/index.tsx [hunk 10]
```diff
@@ -1,34 +1,38 @@
...
+                type={isSubmit ? 'submit' : 'button'}
+                onClick={isSubmit ? undefined : onClick}
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/CustomButton/index.tsx:30]: done

File: src/components/Slider/index.tsx [hunk 11]
```diff
@@ -1,105 +1,130 @@
...
 
 import styles from './styles.module.scss';
 
 interface Props {
-  slides:{id:number, img:string}[];
-  className?:string;
-  auto?:boolean;
-  hide?:boolean
-  setAuto:(auto:boolean)=> void;
+    slides: { id: number; img: string }[];
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/Slider/index.tsx:12]: done

**Review by nathan-hyan — APPROVED** [review]

**Pull request merged**